src/main/java/com/replyyes/facebook/messenger/bean/FacebookMessengerSendException.java

```diff
@@ -0,0 +1,18 @@
+package com.replyyes.facebook.messenger.bean;
+
+import lombok.Getter;
+
+@Getter
+public class FacebookMessengerSendException extends Exception {
+    private static final long serialVersionUID = 5013033180081733121L;
+
+    private final Long errorCode;
+    private final Long errorSubCode;
+
+    public FacebookMessengerSendException(ErrorPayload error) {
```

Comment:
Doesn't work for constructors that call `super`, since that has to be the first line of the constructor. So lombok can't inject the null-check-and-throw bit. The net result is still a null pointer exception, though..